src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilter.java

```diff
@@ -88,24 +94,29 @@ public boolean accept(AuditEvent event) {
     /**
      * To finish the part of this component's setup.
      *
-     * @throws Exception if there is a configuration error.
+     * @throws CheckstyleException if there is a configuration error.
      */
-    protected void finishLocalSetup() throws Exception {
+    protected void finishLocalSetup() throws CheckstyleException {
         if (file != null) {
             loadPatchFile(file);
         }
     }
 
-    private PatchFilterSet loadPatchFile(String patchFileName) throws IOException {
-        final List<String> originPatch = Files.readAllLines(new File(patchFileName).toPath());
-        final List<List<String>> patchList = getPatchList(originPatch);
-        for (List<String> singlePatch : patchList) {
-            final String fileName = getFileName(singlePatch);
-            final Patch<String> patch = UnifiedDiffUtils.parseUnifiedDiff(singlePatch);
-            final List<List<Integer>> lineRangeList = getLineRange(patch);
-            final SuppressionPatchFilterElement element =
-                    new SuppressionPatchFilterElement(fileName, lineRangeList);
-            filters.addFilter(element);
+    private PatchFilterSet loadPatchFile(String patchFileName) {
+        try {
+            final List<String> originPatch = Files.readAllLines(new File(patchFileName).toPath());
+            final List<List<String>> patchList = getPatchList(originPatch);
+            for (List<String> singlePatch : patchList) {
+                final String fileName = getFileName(singlePatch);
+                final Patch<String> patch = UnifiedDiffUtils.parseUnifiedDiff(singlePatch);
+                final List<List<Integer>> lineRangeList = getLineRange(patch);
+                final SuppressionPatchFilterElement element =
+                        new SuppressionPatchFilterElement(fileName, lineRangeList);
+                filters.addFilter(element);
+            }
+        }
+        catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
you can rethrow `CheckstyleException` here

Reply:
done